cve_bin_tool/cli.py

```diff
@@ -349,7 +356,37 @@ def main(argv=None):
         choices=["tag", "json", "yaml"],
         help="specify format of software bill of materials (sbom) to generate (default: tag)",
     )
-
+    output_group.add_argument(
+        "--vex-output",
+        action="store",
+        help="Provide vulnerability exchange (vex) filename to generate",
+        default="",
+    )
+    output_group.add_argument(
+        "--vex-type",
+        action="store",
+        default="cyclonedx",
+        choices=["cyclonedx", "csaf", "openvex"],
+        help="specify type of vulnerability exchange (vex) to generate (default: cyclonedx)",
+    )
+    output_group.add_argument(
+        "--product",
+        action="store",
+        default="",
+        help="Product name for VEX generation",
```

Comment:
Whilst the product name will be used for VEX generation, I think we could also use it for other reports. Suggest you remove 'for VEX Generation' from the help text.